src/features/override-emotes.js

```diff
@@ -32,7 +32,7 @@ module.exports = function() {
                         } else if ($emote.data('channel') && $emote.data('channel') === 'BetterTTV Emotes') {
                             return 'Emote: ' + raw + '<br />BetterTTV Emoticon';
                         } else if ($emote.data('channel')) {
-                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel');
+                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel') + ' (BTTV)';
```

Comment:
You should not worry about about how this data is managed. Every emote that we're aware of is mapped to a channel. If not, it just won't return a channel at all (which is fine).